### PR TITLE
Adding skeleton-archetype

### PR DESCRIPTION
**Pull request by frankyn**
Adding skeleton-archetype with a README describing available archetypes.

Fixes: #17

### CODE REVIEW

File: skeleton-archetype/src/main/resources/archetype-resources/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Google Cloud Platform Archetype List
+
+The following archetypes are provided by Google Cloud Platform for Maven.
+
+* Guestbook Archetype
```
Comment (lesv) [inline skeleton-archetype/src/main/resources/archetype-resources/README.md]: Can you give their `com.google.appengine.archetype` path?

**Review by lesv — APPROVED** [review]
LGTM - with fix to README

File: skeleton-archetype/src/main/resources/archetype-resources/README.md (hunk 1 above)
Comment (frankyn) [reply skeleton-archetype/src/main/resources/archetype-resources/README.md]: Sure, I'll add it for each archetype.

**Comment by frankyn** [comment]
@lesv PTAL

**Review by lesv — APPROVED** [review]

**Pull request merged**